x-pack/plugin/otel-data/src/main/java/org/elasticsearch/xpack/oteldata/OTelPlugin.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.oteldata;
```

Comment:
I'm wondering whether we should call the plugin just `otel`. That may be more future-proof in case we want to also add something like an OTLP endpoint to ES at some point. Having said that, `otel-data` may still be fitting and if not we could also just rename the module.

Reply:
FWIW the "apm-data" plugin was originally just called "x-pack-apm", and was renamed because it introduced confusion with modules/apm. I could see the same thing happening with otel, though maybe if you get in first it doesn't matter ;)